entity-service/src/test/kotlin/com/egm/stellio/entity/authorization/Neo4jAuthorizationRepositoryTest.kt

```diff
@@ -654,51 +648,46 @@ class Neo4jAuthorizationRepositoryTest : WithNeo4jContainer {
 
         assertEquals(2, result.first)
         assertEquals(2, result.second.size)
-        assertTrue {
-            result.second.all {
-                it.type.size == 1 &&
-                    it.right == R_CAN_ADMIN || it.right == AccessRight.R_CAN_WRITE
-            }
-        }
-
-        assertFalse {
-            result.second.all { it.right == AccessRight.R_CAN_READ }
-        }
+        assertTrue(result.second.all { it.type.size == 1 && it.type[0] == "Beekeeper" })
+        assertFalse(result.second.all { it.right == R_CAN_READ })
```

Comment:
does it bring something that is not covered by the following 2 lines?

Reply:
indeed I can remove this one